src/store/post.js

```diff
@@ -101,14 +109,65 @@ const mutations = {
     state.postTitle = data.title
     state.postTags = data.tags || []
     state.blocks = data.blocks || []
-    state.categorySelected = (data.category) ? data.category : '未分類'
-    state.citations = data.citations || []
+    state.categorySelected = data.category ? data.category : '未分類'
+    state.citation = new CitationManager()
```

Comment:
Yes, it will invoke more than once if user write more than one post or edit more than one post at a time.
`reset()` method added to `CitationManager`. It is called in `INIT_POST` and `CitationManager` won't be re-instantiated. Only data will be reset.
When `getDefaultState` is called, it will reset the `CitationManager` too.
Only one `CitationManager` is instantiated in `store/post`
Please have a look.